docs/host-and-deploy/index.md

```diff
@@ -53,7 +53,13 @@ Because browsers make requests to Internet-based hosts for client-side pages, we
 
 The app base path is the virtual app root path on the server. For example, an app that resides on the Contoso server in a virtual folder at `/CoolBlazorApp/` is reached at `https://www.contoso.com/CoolBlazorApp` and has a virtual base path of `/CoolBlazorApp/`. By setting the app base path to `/CoolBlazorApp/`, the app is made aware of where it virtually resides on the server. The app can use the app base path to construct URLs relative to the app root from a component that isn't in the root directory. This allows components that exist at different levels of the directory structure to build links to other resources at locations throughout the app. The app base path is also used to intercept hyperlink clicks where the `href` target of the link is within the app base path URI space&mdash;the Blazor router handles the internal navigation.
 
-In many hosting scenarios, the server's virtual path to the app is the root of the app. In these cases, the app base path is `/`, which is the default configuration for a Blazor app. In other hosting scenarios, such as GitHub Pages and IIS virtual directories, the app base path must be set to the server's virtual path to the app. To set the Blazor app's base path, update the **\<base>** tag in *index.html* on the **\<head>** tag. Change the `href` attribute value from `/` to `/<virtual-path>/` (the trailing slash is required), where `/<virtual-path>/` is the full virtual app root path on the server for the app. 
+In many hosting scenarios, the server's virtual path to the app is the root of the app. In these cases, the app base path is `/`, which is the default configuration for a Blazor app. In other hosting scenarios, such as GitHub Pages and IIS virtual directories, the app base path must be set to the server's virtual path to the app. To set the Blazor app's base path, update the **\<base>** tag in *index.html* on the **\<head>** tag. Change the `href` attribute value from `/` to `/<virtual-path>/` (the trailing slash is required), where `/<virtual-path>/` is the full virtual app root path on the server for the app.
+
+## Static assets
+
+When a client-side Blazor app has access to the Internet, host large static assets (for example, images) on a Content Delivery Network (CDN), such as the [Azure CDN](https://azure.microsoft.com/services/cdn/). By hosting large static assets outside of the app, the app's payload size is reduced.
+
+Alternatively, a client-side Blazor app configures [Static File Middleware](https://docs.microsoft.com/aspnet/core/fundamentals/static-files) to handle static file requests within the app, not from the server project. Static assets can't be served from a Blazor server-side app to a Blazor client-side app outside of the app's initial load from the server. An attempt to add Static File Middleware to the server project with assets in the server project's *wwwroot* folder will fail to serve the assets to the client-side app.
```

Comment:
I'm sure what this is trying to say. I tried adding static file middleware and a wwwroot folder to my server project and the client Blazor app was able to use the static assets from the server project just fine.

Reply:
There may have been updates that made it possible after I tested. I was testing back before May 1, so perhaps things have changed. I'll try again ... *as soon as I can get free.* 😅 🏃